app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     compileSdkVersion 26
     defaultConfig {
         applicationId "ithaca_transit.android.cornellappdev.com.ithaca_transit"
-        minSdkVersion 23
+        minSdkVersion 24
```

Comment:
you should be able to change it bak to 21. Read the notes on the FutureNovaJava repo!

Reply:
Done, thanks for updating the repo!